tools/justfile

```
@@ -70,56 +70,58 @@ _usage:
   EOF
 
 # Custom command to build a single image
-_build-image imageTag dockerFile:
+build imageTag dockerFile:
   docker build -t {{ imageTag }} -f {{ dockerFile }} ../
 
 # Build all images necessary for the deployment 
-_build-armonik-components:
+build-all:
   #!/usr/bin/env bash
   set -euo pipefail
   if [ "x{{ worker }}" == "xstream" ]
   then
-    just _build-image $ARMONIK_WORKER  "../Tests/Stream/Server/Dockerfile" 
+    just build $ARMONIK_WORKER  "../Tests/Stream/Server/Dockerfile"
   elif [ "x{{ worker }}" == "xhtcmock" ]
   then
-    just _build-image $ARMONIK_WORKER "../Tests/HtcMock/Server/src/Dockerfile"
+    just build $ARMONIK_WORKER "../Tests/HtcMock/Server/src/Dockerfile"
   fi
 
-  just _build-image $ARMONIK_METRICS  "../Control/Metrics/src/Dockerfile"
+  just build $ARMONIK_METRICS  "../Control/Metrics/src/Dockerfile"
 
-  just _build-image $ARMONIK_PARTITIONMETRICS "../Control/PartitionMetrics/src/Dockerfile"
+  just build $ARMONIK_PARTITIONMETRICS "../Control/PartitionMetrics/src/Dockerfile"
 
-  just _build-image $ARMONIK_SUBMITTER "../Control/Submitter/src/Dockerfile"
+  just build $ARMONIK_SUBMITTER "../Control/Submitter/src/Dockerfile"
 
-  just _build-image $ARMONIK_POLLINGAGENT "../Compute/PollingAgent/src/Dockerfile" 
+  just build $ARMONIK_POLLINGAGENT "../Compute/PollingAgent/src/Dockerfile"
 
 # Insert partitions in database
-_set-partitions:
+set-partitions:
   docker run --net armonik-backend --rm rtsp/mongosh mongosh mongodb://database:27017/database \
     --eval 'db.PartitionData.insertMany([{ _id: "TestPartition0", ParentPartitionIds: [], PodReserved: 50, PodMax: 100, PreemptionPercentage: 20, Priority: 1, PodConfiguration: null},{ _id: "TestPartition1", ParentPartitionIds: [], PodReserved: 50, PodMax: 100, PreemptionPercentage: 20, Priority: 1, PodConfiguration: null},{ _id: "TestPartition2", ParentPartitionIds: [], PodReserved: 50, PodMax: 100, PreemptionPercentage: 20, Priority: 1, PodConfiguration: null}])'
 
+# Custom compose generic rule
+compose *args:
+  docker-compose -f $COMPOSE_BASE -f $OVERRIDE -f $QUEUE {{args}}
```

I just checked the doc, and it is not the right way to handle args: https://github.com/casey/just#positional-arguments
```suggestion
  docker-compose -f $COMPOSE_BASE -f $OVERRIDE -f $QUEUE "$@"
```